Tests/HummingbirdWebSocketTests/WebSocketExtensionTests.swift

```diff
@@ -300,7 +300,7 @@ struct XorWebSocketExtension: WebSocketExtension {
     func shutdown() {}
 
     func xorFrame(_ frame: WebSocketFrame, context: WebSocketExtensionContext) -> WebSocketFrame {
-        var newBuffer = context.allocator.buffer(capacity: frame.data.readableBytes)
+        var newBuffer = ByteBufferAllocator().buffer(capacity: frame.data.readableBytes)
```

Comment:
Is it possible to this make `func xorFrame(_ frame: inout WebSocketFrame)` and apply the xor on the original frame? It feels wasteful to always force a copy of the data here.

Reply:
Seeing as it's just a test I'm not that fussed.